src/components/rmrk/Gallery/CollectionItem.vue

```diff
@@ -44,11 +48,13 @@
         </div>
       </div>
 
-      <CollectionActivity :nfts="stats" />
+      <CollectionActivity v-if="!statsLoading" :nfts="stats" />
+      <b-skeleton :active="statsLoading" height="30px" :animated="true"></b-skeleton>
```

Comment:
I don't know if this one is really necessary, but I would move the skeleton in CollectionActivity component